src/plugins/workspace/server/workspace_client.ts

```diff
@@ -125,6 +227,52 @@ export class WorkspaceClientWithSavedObject implements IWorkspaceDBImpl {
           type: WORKSPACE_TYPE,
         }
       );
+      const scopedClientWithoutPermissionCheck = this.getScopeClientWithoutPermisson(requestDetail);
```

Comment:
There has been a PR fixing typos, please change the function name to `getScopedClientWithoutPermission`.

Reply:
Fixed and thanks